protocol/errors.json

```diff
@@ -107,6 +107,7 @@
 
   "50010": "Ably's edge proxy service has encountered an unknown internal error whilst processing the request",
   "50210": "Ably's edge proxy service received an invalid (bad gateway) response from the Ably platform",
+  "50305": "Ably's routing layer was unable to service this request",
```

Comment:
Let's add 50205 and 50405 too, which are also set by the router in a similar place to 50305.